ci/tasks/check-canaries-in-error-state.yml

```diff
@@ -0,0 +1,46 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: govukpay/concourse-runner
+params:
+  AWS_ACCESS_KEY_ID:
+  AWS_SECRET_ACCESS_KEY:
+  AWS_SESSION_TOKEN:    
+  AWS_REGION: eu-west-1
+  ENVIRONMENT:
```

Comment:
This should be one of `test`, `stag` or `prod` as that's what the canaries are appended with (due to the character limitation). However I'm aware this makes it confusing for ymls that call this yml. Happy to accept `staging`  and map it to `stag` internally here if people think it's a good idea.

Reply:
I think it's fine - we can always revise later

Reply:
Or maybe `CANARY_POSTFIX`?

Reply:
Nah I think we're good here 😸